.travis.yml

```diff
@@ -24,9 +24,9 @@ matrix:
   include:
     - node_js: "12"
       os: osx
-      # using the default script to build & run the tests
 
     - node_js: "12"
+      # using the default script to build & run the tests
```

Comment:
Please move this comment back to L27. It's saying that the osx job is using the default build & test script.

I find the comment confusing in the context of code-lint job which uses a custom script instead.